From 23e2bf564a1ab65a01eab35189eae40f89368696 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=8D=C3=B1igo=20Huguet?= <ihuguet@redhat.com>
Date: Fri, 21 Jan 2022 08:49:01 +0100
Subject: [PATCH 05/36] rtw89: remove unneeded semicolon
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Bugzilla: http://bugzilla.redhat.com/2033291

commit e0e037b9fe5fbd19b21c0e542e44dd65f1b8cf58
Author: Yang Li <yang.lee@linux.alibaba.com>
Date:   Tue Oct 19 15:54:54 2021 +0800

    rtw89: remove unneeded semicolon

    Eliminate the following coccicheck warning:
    ./drivers/net/wireless/realtek/rtw89/pci.c:1348:2-3: Unneeded semicolon

    Reported-by: Abaci Robot <abaci@linux.alibaba.com>
    Fixes: e3ec7017f6a2 ("rtw89: add Realtek 802.11ax driver")
    Signed-off-by: Yang Li <yang.lee@linux.alibaba.com>
    Acked-by: Ping-Ke Shih <pkshih@realtek.com>
    Signed-off-by: Kalle Valo <kvalo@codeaurora.org>
    Link: https://lore.kernel.org/r/1634630094-1156-1-git-send-email-yang.lee@linux.alibaba.com

Signed-off-by: Íñigo Huguet <ihuguet@redhat.com>
---
 drivers/net/wireless/realtek/rtw89/pci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/realtek/rtw89/pci.c b/drivers/net/wireless/realtek/rtw89/pci.c
index e9731012bca9..2c94762e4f93 100644
--- a/drivers/net/wireless/realtek/rtw89/pci.c
+++ b/drivers/net/wireless/realtek/rtw89/pci.c
@@ -1345,7 +1345,7 @@ static int rtw89_pci_check_mdio(struct rtw89_dev *rtwdev, u8 addr, u8 speed, u16
 	default:
 		rtw89_err(rtwdev, "[ERR]Error Speed %d!\n", speed);
 		return -EINVAL;
-	};
+	}
 	rtw89_write16(rtwdev, R_AX_MDIO_CFG, val);
 	rtw89_write16_set(rtwdev, R_AX_MDIO_CFG, rw_bit);
 
-- 
2.13.6

